upgrades/schema/Version_7_0_20221212160000_add_database_install_time.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pim\Upgrade\Schema;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * @author JM Leroux <jean-marie.leroux@akeneo.com>
+ * @copyright 2022 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+final class Version_7_0_20221212160000_add_database_install_time extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Initialize the database install time if not present';
+    }
+
+    public function up(Schema $schema): void
+    {
+        if ($this->installDataExists()) {
+            $this->addSql('SELECT 1');
+
+            return;
+        }
```

Comment:
You can use skipIf here
```suggestion
        $this->skipIf($this->installDataExists(), 'Install data already exists');
```